

// public class EH1 {
//     public static void main(String[] args){
//         int i = 2;
//         int j=0 ;
//         int nums[] = new int[5];
//         try
//         {
//             j = 18/i;
//             System.out.println(nums[1]);
//             System.out.println(nums[4]);
//         }

//         // we can write multiple catch block to handle many types of exception.
//         catch(ArithmeticException e){
//             System.out.println("Something went gone wrong. " +e);
//         }
//         catch(ArrayIndexOutOfBoundsException e){
//             System.out.println("Something went gone wrong. " +e);
//         }
//         // it should be at the last always.
//         catch(Exception e){
//             System.out.println("Something went gone wrong. " +e);
//         }

//         System.out.println(j);
//     }
// }
